apps/condo/lang/en/en.json

```diff
@@ -2419,12 +2419,12 @@
   "billing.category.heating.name": "Heating",
   "billing.category.membership_fee.name": "Membership fee",
   "billing.category.housing.name.declined": "rent",
-  "billing.category.repair.name.declined": "repairs",
+  "billing.category.repair.name.declined": "capital repairs",
```

Comment:
Why there are duplicated values for `billing.category.repair.name.declined` and `billing.category.repair.name`?

Reply:
they are needed in different cases. In one you need an option with capital letters, in the second an option with lowercase letters